app/View/Users/memberslist.ctp

```diff
@@ -1,4 +1,5 @@
-<div class="users index">
+<div class="users index"><?php
+/*
```

Comment:
please just remove the code if it isn't necessary anymore, we can always take it from the git history if we need it again :)